source/dbo/Functions/Reusable Notification/ufnGetServiceResponsible.sql

```diff
@@ -0,0 +1,37 @@
+﻿CREATE FUNCTION [dbo].[ufnGetServiceResponsible]
+(
+	@TimePeriod int,
```

Comment:
I think `NumberOfYears` would be better than `TimePeriod` as a name for this variable